src/components/Form.jsx

```diff
@@ -1,17 +1,47 @@
-const Form = () => (
-  <div className="form-container">
-    <h2>Add a new book</h2>
-    <form>
-      <input type="text" name="name" placeholder="Book title" />
-      <input type="text" name="author" placeholder="Author" />
-      <select className="selection">
-        <option>Category</option>
-      </select>
-      <button className="add-btn" type="submit">
-        Add
-      </button>
-    </form>
-  </div>
-);
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const Form = () => {
+  const [bookTitle, setTitle] = useState('');
+  const [bookAuthor, setAuthor] = useState('');
+
+  const dispatch = useDispatch();
+
+  const submitForm = (event) => {
+    event.preventDefault();
+    dispatch(addBook({
+      title: bookTitle,
+      author: bookAuthor,
+    }));
+  };
```

Comment:
- [ ] (Optional) Consider setting the value for the title and author to an empty string after submission. For example `setAuthor('')`